paymentsheet/src/main/java/com/stripe/android/link/account/LinkAccountHolder.kt

```diff
@@ -1,16 +1,17 @@
 package com.stripe.android.link.account
 
 import androidx.lifecycle.SavedStateHandle
-import com.stripe.android.link.model.LinkAccount
+import com.stripe.android.link.LinkAccountUpdate
 import kotlinx.coroutines.flow.StateFlow
 
 internal class LinkAccountHolder(
     private val savedStateHandle: SavedStateHandle
 ) {
-    val linkAccount: StateFlow<LinkAccount?> = savedStateHandle.getStateFlow(LINK_ACCOUNT_HOLDER_STATE, null)
+    val linkAccountInfo: StateFlow<LinkAccountUpdate.Value> = savedStateHandle
+        .getStateFlow(LINK_ACCOUNT_HOLDER_STATE, LinkAccountUpdate.Value(null, null))
 
-    fun set(account: LinkAccount?) {
-        savedStateHandle[LINK_ACCOUNT_HOLDER_STATE] = account
+    fun set(info: LinkAccountUpdate.Value) {
+        savedStateHandle[LINK_ACCOUNT_HOLDER_STATE] = info
```

Comment:
Now the LinkAccountHolder stores the latest `LinkAccountUpdate`. That allows us to extend link account info persistence beyond the actual `LinkAccount` model.